%YAML:1.0

image_L_topic: "/cam0/image_raw"
image_R_topic: "/cam1/image_raw"

image_width: 752
image_height: 480

T_BCL: !!opencv-matrix
  cols: 4
  rows: 4
  dt: d
  data: [0.0148655429818, -0.999880929698, 0.00414029679422, -0.0216401454975,
         0.999557249008, 0.0149672133247, 0.025715529948, -0.064676986768,
        -0.0257744366974, 0.00375618835797, 0.999660727178, 0.00981073058949,
         0.0, 0.0, 0.0, 1.0]

intrinsics_L: [458.654, 457.296, 367.215, 248.375] #fu, fv, cu, cv
distortion_coefficients_L: [-0.28340811, 0.07395907, 0.00019359, 1.76187114e-05]

T_BCR: !!opencv-matrix
  cols: 4
  rows: 4
  dt: d
  data: [0.0125552670891, -0.999755099723, 0.0182237714554, -0.0198435579556,
         0.999598781151, 0.0130119051815, 0.0251588363115, 0.0453689425024,
        -0.0253898008918, 0.0179005838253, 0.999517347078, 0.00786212447038,
         0.0, 0.0, 0.0, 1.0]

intrinsics_R: [457.587, 456.134, 379.999, 255.238] #fu, fv, cu, cv
distortion_coefficients_R: [-0.28368365,  0.07451284, -0.00010473, -3.55590700e-05]

baseline: 0.11 # (mm)

num_disparities: 64
P1: 16
P2: 64
Gtau: 8
Q1: 1
Q2: 1
downsample: 1
